src/history.js

```diff
@@ -0,0 +1,3 @@
+import {createBrowserHistory as history} from 'history';
+
+export default history();
```

Comment:
what is this for :0

Reply:
it's to keep track of history when routing pages